things/database.py

```diff
@@ -698,9 +714,9 @@ def make_date_range_filter(date_column: str, offset: str) -> str:
         modifier = f"-{number} years"
 
     column_datetime = f"datetime({date_column}, 'unixepoch', 'localtime')"
-    offset_datetime = f"datetime('now', 'localtime', '{modifier}')"  # type: ignore
```

Comment:
We still "need" the `  # type: ignore` annotation as it's not clear to any static type checker that `modifier` will be set for sure (alternative: having an `else` statement, but it's not really needed).

Reply:
As an aside: I haven't understood the benefit of the static type checker, except for the source code being full of "type: ignore". However, I'm not opposed to it either. Don't hesitate to commit whatever needs changing.